models/DataObject/QuantityValue/Service.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Model\DataObject\QuantityValue;
+
+use Pimcore\Model\Translation;
+
+class Service
+{
+    public static function importDefinitionFromJson(string $json, $override = false): bool
```

Comment:
Add missing type hint
```suggestion
    public static function importDefinitionFromJson(string $json, bool $override = false): bool
```